_test/template.response.test.php

```diff
@@ -0,0 +1,85 @@
+<?php
+//======================================================================
+// TEMPLATE RESPONSE TESTS
+//======================================================================
+
+require_once(__DIR__ . '/../DokuwikiXMLExport.php');
+require_once(__DIR__ . '/../admin.php');
+require_once(__DIR__ . '/../_test/Helper.php');
+
+class template_response_test extends DokuWikiTest
+{
+    public function setUp()
+    {
+        Helper::setUp();
+    }
+
+    /**
+     * This function basically saves a page with a title and checks the response
+     * in admin view of the plugin.
+     */
+    public function test_response_has_translation_set_in_language_file()
+    {
+        $pageHasTitle = 'ihaveatitle';
+        Helper::savePages([$pageHasTitle]);
+        $pageMetaTitle = ['title' => $pageHasTitle];
+        p_set_metadata($pageHasTitle, $pageMetaTitle);
+
+        // Output Buffer
+        ob_start();
+
+        // Call html() function from the plugin
+        $adminPlugin = new admin_plugin_findologicxmlexport();
+        $adminPlugin->html();
+        $output = ob_get_clean();
+        $xml = new SimpleXMLElement($output);
+
+        $header = (string)$xml->xpath('h1')[0];
```

Comment:
It's good practice to add a class or ID to elements. This will make the HTML output more semantic, and the test more robust since you can address the element using the class / ID.